apps/server/src/modules/tldraw/gateway/tldraw.gateway.ts

```diff
@@ -0,0 +1,69 @@
+import { WebSocketGateway, WebSocketServer, OnGatewayInit, OnGatewayConnection } from '@nestjs/websockets';
+import { Server, WebSocket } from 'ws';
+import { encodeStateVector, encodeStateAsUpdate, applyUpdate, Doc } from 'yjs';
+import { MongodbPersistence } from 'y-mongodb-provider';
+import { ConfigService } from '@nestjs/config';
+import { TldrawConfig } from '@src/modules/tldraw/config';
+import { NodeEnvType } from '@src/modules/server';
+import { Configuration } from '@hpi-schul-cloud/commons';
+import { setupWSConnection, setPersistence } from '../utils';
+
+const socketPort: number = Configuration.get('NODE_ENV') === NodeEnvType.TEST ? 3346 : 3345;
```

Comment:
Here, it would be good to have separate ports for development and testing due to conflict if someone would want to start both of them. Socket port is defined in Gateway decorator and config singleton vars is not accessible (maybe I am wrong with this one, this is open for further disscussion)